dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/common/jaxb/JAXBUtils.java

```diff
@@ -1013,7 +1054,9 @@ private static void addToObjectFactoryCache(Package objectFactoryPkg,
             return;
         }
         synchronized (objectFactoryCache) {
-	    LOG.finest("Adding ObjectFactory package to cache: " + objectFactoryPkg); // Liberty Change
+            if (LOG.isLoggable(Level.FINEST)) { 
+                LOG.finest("Adding ObjectFactory package to cache: " + objectFactoryPkg); // Liberty Change
+            }
```

Comment:
No reason for this to be inside the sync block.  Should just move it before the `synchronized (objectFactoryCache) {`